src/Core/ServerSettings.h

```diff
@@ -141,6 +141,8 @@ namespace DB
     M(UInt64, global_profiler_real_time_period_ns, 0, "Period for real clock timer of global profiler (in nanoseconds). Set 0 value to turn off the real clock global profiler. Recommended value is at least 10000000 (100 times a second) for single queries or 1000000000 (once a second) for cluster-wide profiling.", 0) \
     M(UInt64, global_profiler_cpu_time_period_ns, 0, "Period for CPU clock timer of global profiler (in nanoseconds). Set 0 value to turn off the CPU clock global profiler. Recommended value is at least 10000000 (100 times a second) for single queries or 1000000000 (once a second) for cluster-wide profiling.", 0) \
     M(Bool, enable_azure_sdk_logging, false, "Enables logging from Azure sdk", 0) \
+    M(Double, min_os_cpu_wait_time_ratio_to_throw, 2.0, "Min ratio between OS CPU wait and busy time to consider rejecting queries, the probability is 0 at this point", 0) \
+    M(Double, max_os_cpu_wait_time_ratio_to_throw, 10.0, "Max ratio between OS CPU wait and busy time to consider rejecting queries, the probability is 1 at this point", 0) \
```

Comment:
Maybe we can try a smaller default value, like 8

Reply:
Decided to take even lower (6)